.travis.yml

```diff
@@ -1,4 +1,7 @@
 language: python
+os:
+  - linux
+  - osx
```

Comment:
http://docs.travis-ci.com/user/multi-os/#Setting-.travis.yml
But unfortunately they've limited number of osx instance recently to business version.
It's really important to introduce 'osx' environment as I use 'ps auxe' command and 'ps' is not compatible on debian-alike and 'osx' systems. We could use `psutil` library but that's not the direction of mirakuru.